### PR TITLE
API-4550: Retrieve all current API Categories with details

### PR DESCRIPTION
**Pull request by cjrowe**

### CODE REVIEW

File: app/uk/gov/hmrc/apidefinition/controllers/APIDefinitionController.scala [hunk 1]
```diff
@@ -32,6 +32,7 @@ import uk.gov.hmrc.http.{HeaderCarrier, UnauthorizedException}
 import uk.gov.hmrc.play.bootstrap.controller.BackendController
 
 import scala.concurrent.{ExecutionContext, Future}
+import _root_.uk.gov.hmrc.apidefinition.models.APICategory
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/apidefinition/controllers/APIDefinitionController.scala:35]: Remove any _root_ type imports and instead import from correct place - in this case:

`import uk.gov.hmrc.apidefinition.models.APICategory`

File: app/uk/gov/hmrc/apidefinition/models/JsonFormatters.scala [hunk 2]
```diff
@@ -26,7 +26,8 @@ import uk.gov.hmrc.play.json.Union
 
 object JsonFormatters {
 
-  implicit val formatAPICategory = EnumJson.enumFormat(APICategory)
+  // implicit val formatAPICategory = EnumJson.enumFormat(APICategory)
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/apidefinition/models/JsonFormatters.scala:29]: Remove commented out code

**Review by peteslater-ee — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/apidefinition/models/JsonFormatters.scala (hunk 2 above)
Comment (mattclark-zerogravit) [reply app/uk/gov/hmrc/apidefinition/models/JsonFormatters.scala]: Have fixed this

File: app/uk/gov/hmrc/apidefinition/controllers/APIDefinitionController.scala (hunk 1 above)
Comment (mattclark-zerogravit) [reply app/uk/gov/hmrc/apidefinition/controllers/APIDefinitionController.scala]: I have resolved this

**Review by peteslater-ee — APPROVED** [review]

**Pull request merged**